workers/unity/Packages/com.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -66,8 +66,8 @@ public bool TryGetEntity(EntityId entityId, out Entity entity)
 
                 var worker = new Worker(config.WorkerType, connection, logger, origin);
                 logger.HandleLog(LogType.Log, new LogEvent("Successfully created a worker")
-                    .WithField("WorkerType", worker.WorkerType)
-                    .WithField("WorkerId", worker.WorkerId));
+                    .WithField("WorkerId", worker.WorkerId)
+                    .WithField(LoggingUtils.WorkerType, worker.WorkerType));
```

Comment:
Why add this back in if its automatically added?

Reply:
In this instance, Worker is still null. So I pass the name in manually

Reply:
Oops this one doesn't need it.